ILTransform/ILRewriter.cs

```diff
@@ -97,6 +97,24 @@ private void RewriteFile(string source)
                 int lineIndex = _testProject.LastMainMethodDefLine;
                 if (lineIndex >= 0)
                 {
+                    if (_settings.CollapseMainSign
```

Comment:
curious - why is this needed?  If we have the bounds of the definition (first and last), then why can't we just add 'public' and be done?

I'm not necessarily against it, but every transformation has some risk of going wrong.

Reply:
Should that be "CollapseMainLines"?

Reply:
I haven't checked other regexes, but this makes the scenario for them easier to fix and understand. There are not so many cases of main signs on multiple lines ,<10 in JIT/Regression